test/SlowTests/Issues/RavenDB-23136.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using FastTests;
+using FastTests.Utils;
+using Raven.Client.Documents.Operations.Revisions;
+using Raven.Client.Documents.Smuggler;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_23136 : RavenTestBase
+    {
+        public RavenDB_23136(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        [RavenFact(RavenTestCategory.Smuggler | RavenTestCategory.Revisions)]
+        public async Task Database_Shouldnt_Create_Redundant_Revisions_On_Document_Import()
+        {
+            DoNotReuseServer();
+            var file = GetTempFileName();
+            try
+            {
+                await Database_Shouldnt_Create_Redundant_Revisions_On_Document_Import(file);
+            }
+            finally
+            {
+                File.Delete(file);
+            }
+        }
+
+        public async Task Database_Shouldnt_Create_Redundant_Revisions_On_Document_Import(string file)
```

Comment:
if this is a test then it is missing `[RavenFact]`
if this is a method then it should be private and also it could be merged inside the test